### DISCUSSION TITLE
How to prevent browser back button after logout.

### DISCUSSION THREAD

**Discussion by anandhuremanan** [sveltejs/kit#12760] [Q&A]
Hi there,

how do I prevent back button of the browser to load back to the previous route(protected route) after logging out of the app?

Thanks

**Comment by brunnerh** [comment]
I would probably try to redirect from a [`handle` hook](https://kit.svelte.dev/docs/hooks#server-hooks-handle).
Or try utilizing one of the navigation events from [`$app/navigation`](https://kit.svelte.dev/docs/modules#$app-navigation) (`onNavigate`/`beforeNavigate`).
